src/base/dof_map_constraints.C

```diff
@@ -3557,7 +3557,6 @@ void DofMap::scatter_constraints(MeshBase & mesh)
       pushed_keys_vals_to_me[pid] = data;
     };
 
-  // Trade pushed dof constraint rows
```

Comment:
Why'd this vanish?

Reply:
Well - I did have some back and forth on revisions that edited this part of code - just a messup on my part to leave this like this though.  I'll fix it.